spring-data-jpa/src/test/java/org/springframework/data/jpa/repository/query/QueryEnhancerUnitTests.java

```diff
@@ -179,6 +179,17 @@ void findsExistingOrderByIndependentOfCase() {
 		assertThat(query).endsWithIgnoringCase("ORDER BY p.firstname, p.lastname asc");
 	}
 
+	@Test // GH-3263
+	void doesNotDropWithExpressionWhenApplySort() {
```

Comment:
It's better to use `Sorting` instead of `Sort` for consistency.

Reply:
thanks @quaff - additionally I'm not happy with the `doesNotDrop` part. will rename the method.